cfpb/consumerfinance.gov
# Upgrading psycopg2 to work with python 3.9

Upgrading psycog2 to a version that works with our current python version (3.6.9) and the version of python we wish to upgrade to (3.9.9)
---

## Changes

- version upgrade from 2.7.3.2 to 2.8.6


## How to test this PR

1.Clone the repo
2.Using pyenv change your shell's python to version 3.9.9
3. run ip install -r requirements/local.txt in the cloned repo and there should be no problems
4.If you ran into an ssl issue while trying to get the psycog2 version run these commands
`brew install openssl`
`export LDFLAGS="-L$HOME/homebrew/Cellar/openssl@3/3.0.1/lib"`
`export CPPFLAGS="-I$HOME/homebrew/Cellar/openssl@3/3.0.1/include"`
5.The code wouldn't build for the PR checks if the upgrade wasn't also compatible with 3.6.9


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
